### PR TITLE
Add Langfuse as hard

### PR DESCRIPTION
**Pull request by Kefaku**
May actually be limited difficulty instead, needs verification.

### CODE REVIEW

File: _data/sites.json [hunk 1]
```diff
@@ -11617,6 +11617,19 @@
         ]
     },
 
+    {
+        "name": "Langfuse",
+        "url": "https://langfuse.com/support",
+        "difficulty": "hard",
+        "email": "security@langfuse.com",
+        "notes" : "Contact [security@langfuse.com](mailto:security@langfuse.com) and ask for your account to be deleted.",
```
Comment (tupaschoal) [inline _data/sites.json:11625]:
```suggestion
        "notes" : "Contact them via e-mail and ask for your account to be deleted.",
```
We don't repeat the e-mail on the notes.

**Review by tupaschoal — COMMENTED** [review]
Why is it maybe limited? Because you don't know whether it worked yet?

**Comment by Kefaku** [comment]
> Why is it maybe limited? Because you don't know whether it worked yet?
@tupaschoal 

I selected the EU data region for my account. I'm unsure whether that affects the process.

**Comment by tupaschoal** [comment]
Ok, will leave it at hard for the moment, thanks!

**Pull request merged**